tests/utils/test_kms.py

```
@@ -0,0 +1,227 @@
+
+    def mock_hmget(key, fields):
+        result = {}
+        if key in cache._hash_store:
+            for field in fields:
+                result[field] = cache._hash_store[key].get(field)
+        else:
+            for field in fields:
+                result[field] = None
+        return result
+
+    def mock_hmset(key, mapping, ttl=None):  # pylint: disable=unused-argument
```

Ruff (0.14.13)

Unused function argument: `ttl`

(ARG001)